src/collections/configure/unit.test.ts

```diff
@@ -597,6 +597,24 @@ describe('Unit testing of the vectorizer factory class', () => {
         config: {
           deploymentId: 'deployment-id',
           resourceName: 'resource-name',
+          isAzure: true,
+        },
+      },
+    });
+  });
+
+  it('should create the correct Text2VecAzureOpenAIConfig type with just isAzure: true', () => {
+    const config = configure.vectorizer.text2VecAzureOpenAI({});
+    expect(config).toEqual<VectorConfigCreate<never, undefined, 'hnsw', 'text2vec-azure-openai'>>({
+      name: undefined,
+      vectorIndex: {
+        name: 'hnsw',
+        config: undefined,
+      },
+      vectorizer: {
+        name: 'text2vec-azure-openai',
+        config: {
+          isAzure: true,
```

Comment:
Is the output the same if a user supplies `configure.vectorizer.text2VecAzureOpenAI()` without an empty object argument?

Reply:
Yes, it should be - I adjusted the types to allow no config object to be passed 👍